web/app/css/styles.css

```diff
@@ -74,6 +74,14 @@ h2, h3, h4, h5, h6 {
   float: right;
 }
 
+/* center the spinner */
```

Comment:
not sure if this behavior already existed, spinner appears centered over page content, but not window:
![screen shot 2018-07-03 at 4 12 00 pm](https://user-images.githubusercontent.com/236915/42248696-d9642f3a-7edb-11e8-88a2-3c1914bedd24.png)

Reply:
yeah, this was the existing behaviour!